inst/WORDLIST

```diff
@@ -289,6 +306,7 @@ jsonlite
 knitr
 Kubernetes
 lastPathElement
+LateX

```

Comment:
same here. please remove the entry from the wordlist and use `LaTeX ` in the text (capital T)